### PR TITLE
Add support for text2vec-cohere module

### PR DESCRIPTION
**Pull request by antas-marcin**

### CODE REVIEW

File: .cicd/test.sh [hunk 1]
```diff
@@ -104,6 +104,7 @@ function check_creates_template() {
   check_modules "--set modules.text2vec-contextionary.enabled=false --set modules.text2vec-transformers.enabled=false --set modules.img2vec-neural.enabled=false --set modules.qna-transformers.enabled=false --set modules.text-spellcheck.enabled=false --set modules.ner-transformers.enabled=false --set modules.multi2vec-clip.enabled=false --set modules.text2vec-openai.enabled=true --set modules.text2vec-openai.apiKey=apiKey" "value: text2vec-openai"
   check_modules "--set modules.text2vec-contextionary.enabled=true --set modules.sum-transformers.enabled=true" "value: text2vec-contextionary,sum-transformers"
   check_modules "--set modules.text2vec-contextionary.enabled=true --set modules.ref2vec-centroid.enabled=true" "value: text2vec-contextionary,ref2vec-centroid"
+  check_modules "--set modules.text2vec-contextionary.enabled=true --set modules.text2vec-cohere.enabled=true" "value: text2vec-contextionary,text2vec-cohere"
```
Comment (StefanBogdan) [inline .cicd/test.sh:107]: Maybe we should add a test with only `cohere` enabled. Just to make sure the c11y does not interfere with it.

**Review by StefanBogdan — CHANGES_REQUESTED** [review]
I left only one change request.

File: .cicd/test.sh (hunk 1 above)
Comment (antas-marcin) [reply .cicd/test.sh:107]: sure. on it

**Review by StefanBogdan — APPROVED** [review]
Perfect!

**Pull request merged**